tasks/zadanie_21.php

```diff
@@ -44,3 +44,57 @@
 $products = $params[0]; // tej linijki nie ruszamy :)
 $key = $params[1]; // tej linijki nie ruszamy :)
 $sort = $params[2]; // tej linijki nie ruszamy :)
+
+$errorCheck_ = 0;
```

Comment:
:information_source: z takich dobryk praktyk :D ogólnie to raczej nie dajemy _ na końcu zmiennych, nie jest to błąd - bo wszystko działa :D na przyszłość tylko warto wiedzieć że dobrą praktyką będzie raczej cammelCase ;)
```suggestion
$errorCheck = 0;
```